app/javascript/controllers/projects/samples/attachments/concatenations_controller.js

```diff
@@ -0,0 +1,48 @@
+import { Controller } from "@hotwired/stimulus";
+
+export default class extends Controller {
+  static targets = ["field"];
+
+  static values = {
+    fieldName: String,
+    storageKey: {
+      type: String,
+      default: location.protocol + "//" + location.host + location.pathname,
```

Comment:
Can you update this here (and in the selection_controller) to use string templates:
```javascript
default: `${location.protocol}//${location.host}${location.pathname}`
```

Reply:
@joshsadam since you are working on the selection controller in your pr let's just updated this in there

Reply:
Assuming you're going to take care of the selection controller, see 3ee83cb5365b6845c7ab72299cfab40c81b7543f.